x/ibc/09-localhost/client/cli/tx.go

```diff
@@ -23,10 +23,9 @@ func GetCmdCreateClient(cdc *codec.Codec) *cobra.Command {
 	cmd := &cobra.Command{
 		Use:   "create",
 		Short: "create new localhost client",
-		Long: strings.TrimSpace(fmt.Sprintf(`create new localhost (loopback) client:
-
-Example:
-$ %s tx ibc client localhost create --from node0 --home ../node0/<app>cli --chain-id $CID
+		Long: strings.TrimSpace(fmt.Sprintf(`create new localhost (loopback) client:	
```

Comment:
This entire command can be deleted correct?

Reply:
I left it in case the app `createLocalhost` was `false` and someone wanted to create one after initialization.

Reply:
I think if the chain starts without a localhost it probably shouldn't have one. Not sure if localhost client should be added by any user if the chain decides to start without it.

THink it makes more sense to have that be a governance decision by the chain validators and hard-fork if they want to add/remove their chain's localhost

Reply:
Kinda agree. Like it on by default, but if it's off should be disabled.

Reply:
Maybe this should be field on the ibc GenesisState then (enabled by default)? So that apps can configure it on the JSON instead? Thoughts? cc: @cwgoes

Reply:
Agreed with putting it in the `GenesisState`.